configs/ote/face-detection/face-detection-0200/model.py

```diff
@@ -1,15 +1,16 @@
+_base_ = [
+    './coco_data_pipeline.py'
+]
```

Comment:
No need to keep data pipeline in a separate file anymore.

I also wonder, why have you changed the data pipeline at all?

Reply:
We made it w.r.t. the changes that were in model.py earlier

Reply:
This file is removed from the current PR